src/css/imports/controlbar.less

```diff
@@ -153,9 +153,11 @@
 .jw-flag-autostart,
 .jw-flag-user-inactive.jw-state-playing,
 .jw-flag-user-inactive.jw-state-buffering {
-  &:not(.jw-flag-media-audio):not(.jw-flag-audio-player):not(.jw-flag-ads-vpaid-controls):not(.jw-flag-casting) {
+  &:not(.jw-flag-media-audio):not(.jw-flag-audio-player):not(.jw-flag-ads-vpaid-controls):not(.jw-flag-casting):not(.jw-flag-ads) {
```

Comment:
You can't add `:not(.jw-flag-ads)`. That will increase specificity which could introduce regressions. There already is going to be a rule for ads that you need to modify.